e2e/typings/es6-shim/es6-shim.d.ts

```diff
@@ -0,0 +1,670 @@
+// Compiled using typings@0.6.10
```

Comment:
Why do we have to copy these typings into our project instead of installing them?